src/main/java/nl/tudelft/ti2206/group9/util/Writer.java

```diff
@@ -6,12 +6,14 @@
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.OutputStreamWriter;
+import java.io.UnsupportedEncodingException;
 
 import nl.tudelft.ti2206.group9.level.State;
 import nl.tudelft.ti2206.group9.util.GameObserver.Category;
 import nl.tudelft.ti2206.group9.util.GameObserver.Error;
 
 import org.json.simple.JSONObject;
+import sun.misc.BASE64Encoder; //NOPMD - I need this package
```

Comment:
Add @SuppressWarnings("restriction") for using this import.

<review,organization